packages/headless/src/features/generated-answer/generated-answer-analytics-actions.ts

```diff
@@ -49,19 +49,17 @@ export const logOpenGeneratedAnswerSource = (
   makeAnalyticsAction({
     prefix: 'analytics/generatedAnswer/openAnswerSource',
     __legacy__getBuilder: (client, state) => {
-      const {id: rgaID, answerAPIEnabled} =
+      const {generativeQuestionAnsweringId} =
```

Comment:
I don't even know why that's an object :) 
This is the only prop in the object that we return.